projects/subject-access-requests-and-delius/src/main/kotlin/uk/gov/justice/digital/hmpps/integration/delius/person/entity/Person.kt

```diff
@@ -0,0 +1,48 @@
+package uk.gov.justice.digital.hmpps.integration.delius.person.entity
+
+import jakarta.persistence.Column
+import jakarta.persistence.Entity
+import jakarta.persistence.Id
+import jakarta.persistence.Table
+import org.hibernate.annotations.Immutable
+import org.hibernate.annotations.SQLRestriction
+import org.springframework.data.jpa.repository.JpaRepository
+import uk.gov.justice.digital.hmpps.exception.NotFoundException
+
+@Immutable
+@Entity
+@Table(name = "offender")
+@SQLRestriction("soft_deleted = 0")
+class Person(
+    @Id
+    @Column(name = "offender_id")
+    val id: Long,
+
+    @Column(columnDefinition = "char(7)")
+    val crn: String,
+
+    @Column(name = "first_name", length = 35)
+    val forename: String,
+
+    @Column(name = "second_name", length = 35)
+    val secondName: String? = null,
+
+    @Column(name = "third_name", length = 35)
+    val thirdName: String? = null,
+
+    @Column(name = "surname", length = 35)
+    val surname: String,
+
+    @Column(columnDefinition = "number")
+    val softDeleted: Boolean = false,
+) {
+    fun getName(): String {
+        return "$forename " + listOfNotNull(secondName, thirdName).joinToString(" ") + " $surname"
+    }
```

Comment:
If they have no middle names, you'll end up with a double space between forename and surname here.  Might be easier to include forename and surname in the `listOfNotNull` so all the parts are joined in the same way?
```suggestion
    fun getName() = listOfNotNull(forename, secondName, thirdName, surname).joinToString(" ")
```

Reply:
done